examples/bootstrapping/vagrant/onboarding/client/provision.sh

```diff
@@ -0,0 +1,99 @@
+CHEF_WORKSTATION_VERSION=$1
+OS_TYPE=$2
+OS_VERSION=$3
+
+echo "************************"
+echo "Preflight Configuration "
+echo "************************"
+
+sudo sysctl -w vm.max_map_count=262144
+sudo sysctl -w vm.dirty_expire_centisecs=20000
+sudo echo "vm.max_map_count=262144" >> /etc/sysctl.conf
+sudo echo "vm.dirty_expire_centisecs=20000" >> /etc/sysctl.conf
+
+echo "******************************************"
+echo "Downloading & Installing Chef Workstation "
+echo "******************************************"
+
+if [[ $OS_TYPE == *'ubuntu'* ]]
+then
+  #https://packages.chef.io/files/stable/chef-workstation/20.8.111/ubuntu/16.04/chef-workstation_20.8.111-1_amd64.deb
```

Comment:
Can probably remove these comments?  Assuming they were just for testing.

Reply:
Removed.